modules/contact/test/tests/bouncing-block-contact/grid-sequencing/tests

```diff
@@ -0,0 +1,24 @@
+[Tests]
+  design = FEProblemBase.md
+  issues = '#14166'
+  [grid_sequencing]
+    type = Exodiff
+    input = grid-sequencing.i
+    exodiff = grid-sequencing_exo.e
+    requirement = 'The system shall use grid sequencing in order to improve the performance of the nonlinear solve in a frictional contact problem'
+    cli_args = 'Postprocessors/active=contact'
+    # Fix below when I understand duplicate element centroids
+    map = False
+    partial = True
+  []
+  [no_grid_sequencing]
+    type = Exodiff
+    input = no-grid-sequencing.i
+    exodiff = no-grid-sequencing_exo.e
+    requirement = 'The system shall be able to produce the same result when not using grid sequencing'
+    cli_args = 'Postprocessors/active=contact'
```

Comment:
You could `cli_args` to adjust  `grid_sequencing.i` here so that we can delete `no-grid-sequencing.i`